### DISCUSSION TITLE
How can I highlight html tags on jsx ?

### DISCUSSION THREAD

**Discussion by luisiacc** [nvim-treesitter/nvim-treesitter#1007] [Q&A]
So I have something like this:
```jsx
    <Layout home>
      <Head>
        <title>{siteTitle}</title>
      </Head>
	</Layout>

```

When I run TSHighlightCapturesUnderCursor on `title`, says `@TSTag` and `@TSVariable` , however highlighting `TSTag` does match correctly agains `Layout` and `Head`, but it does not for `title`, which is getting the highlight from `TSVariable`, so my question would be how to properly highlight the `title` tag as a tag, because highlighting `TSVariable` will also highlight all variables.

**Comment by kyazdani42** [comment]
this is an issue that has been there for quite some time, here #921
